python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -0,0 +1,33 @@
+###########################################################################################################
+#This bash script is meant to support prometheus metrics collection via cloudwatch agent.
+#You can view the ray metrics being collected through AWS CloudWatch console.
+###########################################################################################################
+
+read -r -p "please specify your CLUSTER_CONFIG_FILE to start a cluster" AUTOSCALER_CONFIG_FILE
```

Comment:
Minor: We could make this a bit simpler and also support optional `ray up` parameters by just passing all input arguments provided to this script over to `ray up`.